src/js/navigationtoggle.js

```diff
@@ -8,8 +8,6 @@ const butBook = document.querySelector('[data-js=buttonBook]')
 const butCreate = document.querySelector('[data-js=buttonCreate]')
```

Comment:
I would suggest renaming this to `createButton` or `buttonCreate` as it is easier to read and we should name the things as they are. At first I thought _why is he referring to a but?_ and only noticed that it is a button after finding the selector.
```suggestion
const buttonCreate = document.querySelector('[data-js=buttonCreate]')
```